### PR TITLE
Add Pulsar Community Weekly Update - 2020-11-07~ 2020-11-13 and Update Weekly 1106

### PR DESCRIPTION
**Pull request by Huanli-Meng**

### CODE REVIEW

File: content/2020/2020-11/2020-11-13-pulsar-weekly.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+  -  Pulsar Summit Asia 2020
+
+     https://pulsar-summit.org/en/event/asia-2020
+
+### Blog / Article
+
+- Event-driven railway network based on Pulsar -- Pavels Sisojevs
+
+  - https://streamnative.io/blog/release/2020-10-28-streamnative-free-cloud
```
Comment (Jennifer88huang) [inline content/2020/2020-11/2020-11-13-pulsar-weekly.md:137]: remove it?

File: content/2020/2020-11/2020-11-13-pulsar-weekly.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+    https://github.com/apache/pulsar/pull/8474 (@[315157973](https://github.com/315157973))
+
+- [Function] Enable E2E encryption for Pulsar Function.
+
+    https://github.com/apache/pulsar/pull/8432 (@[nlu90](https://github.com/nlu90))
+
+### Notable Bug Fix
+
+- [Functions] Fix the issue that when overriding `jobName` in Kubernetes runtime with custom runtime options can have collisions when same `jobName` is used.
```
Comment (Jennifer88huang) [inline content/2020/2020-11/2020-11-13-pulsar-weekly.md:93]: This PR title needs refinement.

File: content/2020/2020-11/2020-11-13-pulsar-weekly.md (hunk 1 above)
Comment (Huanli-Meng) [reply content/2020/2020-11/2020-11-13-pulsar-weekly.md]: updated

File: content/2020/2020-11/2020-11-13-pulsar-weekly.md (hunk 2 above)
Comment (Huanli-Meng) [reply content/2020/2020-11/2020-11-13-pulsar-weekly.md]: updated

**Review by Jennifer88huang — APPROVED** [review]

**Pull request merged**